src/vivarium_census_prl_synth_pop/data/loader.py

```diff
@@ -307,3 +292,33 @@ def create_draws(df: pd.DataFrame, key: str, location: str):
     )
 
     return draws
+
+
+#####################
+#  Helper functions #
+#####################
+
+
+def _read_and_format_raw_data(
+    data_dir: Path, filenames: List[str], column_map: List[str], location: str
+) -> pd.DataFrame:
+    data = pd.concat(
+        [pd.read_csv(data_dir / file, usecols=column_map.keys()) for file in filenames]
+    )
+
+    data["SERIALNO"] = data["SERIALNO"].astype(str)
+    data = data.rename(columns=column_map)
+
+    if location == "United States of America":
+        location_ids = [
+            v
+            for k, v in metadata.CENSUS_STATE_IDS.items()
+            if k in metadata.UNITED_STATES_LOCATIONS
+        ]
+        data = data.query(f"state in {location_ids}")
```

Comment:
I think in general we don't use `query`, but I see that you based this on existing code using `query` so it's not a big deal either way.

Reply:
Yeah I personally hate `query` and initially wrote this with standard slicing. But then I switched for consistency

Reply:
It's fine not to use `query`, btw.